bag_path: '/home/brik/Desktop/2023-10-04-17-14-02_filtered_cropped.bag'
imu_topic: '/imu/data_raw'
radar_topic: '/radar/cfar_detections'
odometry_topic: '/rio/odometry_optimizer'

B_t_BR: [0.115, -0.002, -0.018]      # Initial guess position of radar in body frame [x, y, z]
q_BR: [0.705, 0.705, -0.049, -0.049]  # Initial guess orientation of radar in body frame [x, y, z, w]